src/v2/Components/UserSettings/TwoFactorAuthentication/Components/BackupSecondFactor/BackupSecondFactorModalContent.tsx

```diff
@@ -33,6 +34,9 @@ export const BackupSecondFactorModalContent: React.FC<BackupSecondFactorModalCon
           </Box>
         ))}
       </Flex>
+
+      <BackupSecondFactorActions
+        backupSecondFactors={me.backupSecondFactors!.map(item => item!.code!.toString())} />
```

Comment:
I guess it's safe to assume the presence of these elements, right?

Reply:
Upon initial mutation, we would handle with the APIErrorModal [here](https://github.com/artsy/force/blob/master/src/v2/Components/UserSettings/TwoFactorAuthentication/Components/SmsSecondFactor/Modal.tsx#L77) and so not reach the Recovery Code modal without the backup codes having been created. 

Took a quick look but it is less clear when the user is just checking their already existing backup codes. Would be good to verify and handle gracefully if we aren't already.